GPU/GLES/TextureCacheGLES.cpp

```diff
@@ -483,7 +487,7 @@ void TextureCacheGLES::ApplyTextureFramebuffer(TexCacheEntry *entry, VirtualFram
 
 		framebufferManagerGL_->BindFramebufferAsColorTexture(0, framebuffer, BINDFBCOLOR_SKIP_COPY);
 		render_->BindTexture(TEX_SLOT_CLUT, clutTexture);
-		render_->SetTextureSampler(TEX_SLOT_CLUT, GL_CLAMP_TO_EDGE, GL_CLAMP_TO_EDGE, GL_NEAREST, GL_NEAREST, 0.0f);
+		render_->SetTextureSampler(TEX_SLOT_CLUT, GL_REPEAT, GL_CLAMP_TO_EDGE, GL_NEAREST, GL_NEAREST, 0.0f);
```

Comment:
Hm, does it matter how we access the CLUT (do we go out of bounds somehow)?

Reply:
I just copied this from before - it didn't fix it but I figured I'd keep it.

The actual PSP does wrap access to the CLUT (in the case of using an offset.)  It's an edge case for sure.

-[Unknown]